server/store.go

```diff
@@ -0,0 +1,250 @@
+package main
+
+import (
+	"database/sql"
+
+	sq "github.com/Masterminds/squirrel"
+	_ "github.com/go-sql-driver/mysql"
+	_ "github.com/lib/pq"
+	"github.com/mattermost/mattermost-server/v5/model"
+)
+
+const (
+	ActivePeriodInMinutes = 7 * 24 * 60 // A week
+)
+
+type DBStore struct {
+	conn *sql.DB
+	sq   sq.StatementBuilderType
+}
+
+func NewDBStore(driverName, dataSource string) (*DBStore, error) {
+	db, err := sql.Open(driverName, dataSource)
```

Comment:
May be it's better to use `mattermost-plugin-api` to get the db, it will manage the number of connections.

Reply:
Would be better, but the last tagged version of mattermost-plugin-api doesn't have still the Store services. I would wait until that is tagged and then we can use it.